CHANGELOG.md

```diff
@@ -2,16 +2,20 @@
 
 ## Unreleased
 
+## v0.45.0 Beta
+
 ### 🛑 Breaking changes 🛑
 
 - Remove deprecated funcs in configtelemetry (#4808)
 - Deprecate `service/defaultcomponents` go package (#4622)
+- `otlphttp` and `otlp` exporters enable gzip compression by default (#4632)
```

Comment:
Looks like this was incorrectly added to 0.44.0, right?

Reply:
Correct, I moved them here to reflect the right version they got in.